### PR TITLE
Bug Fix: Tests

### PR DESCRIPTION
**Pull request by itay-grudev**

### CODE REVIEW

File: charts/cluster/test/paradedb-minio-backup-restore/01-paradedb_cluster.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 type: paradedb
 mode: standalone
+version:
+  paradedb: 0.9.4
+
```
Comment (itay-grudev) [inline charts/cluster/test/paradedb-minio-backup-restore/01-paradedb_cluster.yaml:5]: Could be removed when ran with your workflow in-place modification, but I feel this is better.

File: charts/cluster/README.md [hunk 2]
```diff
@@ -20,6 +20,22 @@ cnpg/cloudnative-pg
 
 ### Setting up a ParadeDB CNPG Cluster
 
+Create a `values.yaml` and configure it to your requirements. Here is a basic example:
+
+```yaml
```
Comment (philippemnoel) [inline charts/cluster/README.md:25]: @itay-grudev I thought the point of the values.yaml file we have in the repo is so this is not needed? Can't we use the existing `values.yaml`?

Comment (itay-grudev) [reply charts/cluster/README.md:25]: Those are the chart defaults. This is the user's own configuration.

Comment (philippemnoel) [reply charts/cluster/README.md:25]: Why does the user need to provide their own configuration? Can't it inherit from the chart's defaults?

File: charts/cluster/test/paradedb-minio-backup-restore/01-paradedb_cluster.yaml (hunk 1 above)
Comment (philippemnoel) [reply charts/cluster/test/paradedb-minio-backup-restore/01-paradedb_cluster.yaml:5]: Why do we need a version? As far as I can tell the other cluster types don't have a version specified

File: charts/cluster/README.md [hunk 3]
```diff
@@ -20,6 +20,22 @@ cnpg/cloudnative-pg
...
+  instances: 2
+  storage:
+    size: 256Mi
+```
+
+You can refer to the other examples in the [`charts/cluster/examples`](https://github.com/paradedb/charts/tree/main/charts/cluster/examples) directory.
+
+```yaml
+
 ```console
```
Comment (philippemnoel) [inline charts/cluster/README.md:39]: Maybe we can just make this omit the `values.yaml`

**Review by philippemnoel — APPROVED** [review]

**Pull request merged**